__tests__/app.test.js

```diff
@@ -232,3 +233,75 @@ describe("Patch /api/reviews/:review_id", () => {
     return request(app).patch("/api/reviews/1").expect(400);
   });
 });
+
+describe("GET /api/reviews/:review_id/comments", () => {
+  test("returns an array of objects and a 200 status", () => {
+    return request(app)
+      .get("/api/reviews/2/comments")
+      .expect(200)
+      .then((res)=>{
+        expect(res.body.data).toBeInstanceOf(Array)
```

Comment:
comments should be returned on a key of comments! So `{ comments: [ {...}, ...] }